plugins/naturalspeech

```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/phyce/rl-natural-speech.git
+commit=45e124a46608daccc27a3a397a56aa61bc2f22e0
+warning=This plugin downloads voice model files from external services.
```

Comment:
```suggestion
warning=This plugin downloads voice model files from huggingface.co, a 3rd party website not controlled or verified by the RuneLite Developers.
```
basically all the warnings follow the same [structure](https://github.com/search?q=repo%3Arunelite%2Fplugin-hub+warning%3D&type=code)

Reply:
As felanbird said, please use the same format used by most/all other plugins with warning fields. Additionally, please do not exclude the warning about IP addresses being sent. osrs users are paranoid, and may be upset to learn this some other way.

Reply:
Yeah, that is absolutely fair. I've commited the suggestion.
I am looking at traing custom models eventually, and might end up just hosting those along with these all on a repo on github. If the models are hosted in github will it still need the warning?

Reply:
No, there's a short list of servers/services we trust, and GitHub is one of them.